reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    solve_energy = "adiabatic"
</pre><p>I think this should be a Boolean (<code>True</code> / <code>False</code> )  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Modified this variable to a Boolean.</p>